packages/button/src/IconButton.tsx

```diff
@@ -14,7 +14,7 @@ import { isValidElement, cloneElement, forwardRef, memo } from 'react';
 import './styles/Button.css';
 
 type IconButtonProps = ButtonHTMLAttributes<HTMLButtonElement> & {
-  kind?: 'default' | 'primary' | 'destructive' | 'minimal' | 'close';
+  kind?: 'basic' | 'primary' | 'destructive' | 'minimal' | 'close';
```

Comment:
The "default" kind was only the actual default for Buttons, so `basic` is a better name.